examples/phabricator/README.md

```diff
@@ -43,16 +43,36 @@ This example assumes that you have a basic understanding of kubernetes [services
 
 ```sh
 $ cd kubernetes
-$ hack/dev-build-and-up.sh
+$ cluster/kube-up.sh
 ```
 
 ### Step One: Set up Cloud SQL instance
 
 Follow the [official instructions](https://cloud.google.com/sql/docs/getting-started) to set up Cloud SQL instance.
 
-In the remaining part of this example we will assume that your instance is named "phabricator-db", has IP 173.194.242.66 and the password is "1234".
+In the remaining part of this example we will assume that your instance is named "phabricator-db", has IP 173.194.105.37, is listening on port 3306 and the password is "1234".
```

Comment:
it would be easier to read this if you made up an IP that was easy to remember :)